site/en/docs/extensions/reference/storage/index.md

```diff
@@ -44,7 +44,13 @@ Local and sync storage areas should not store confidential user data because the
 {% endAside %}
 
 [storage.session][prop-session]
-: Holds data in memory for the duration of a browser session. By default, it's not exposed to content scripts, but this behavior can be changed by setting [`chrome.storage.session.setAccessLevel()`][method-access-level]. The quota limitation is 1 MB approx. Consider using it to store global variables across service worker runs.
+: Holds data in memory for the duration of a browser session. By default, it's not exposed to content scripts, but this behavior can be changed by setting [`chrome.storage.session.setAccessLevel()`][method-access-level]. The quota limitation is approximately 10 MB. Consider using it to store global variables across service worker runs.
+
+{% Aside 'warning' %}
+
+Before Chrome 112, the quota was approximately 1 MB.
+
+{% endAside %}
```

Comment:
```suggestion
{% Aside 'warning' %}
Before Chrome 112, the quota was approximately 1 MB.
{% endAside %}
```

Reply:
I actually don't remember if the widget honors that spacing. The point is that the widget controls it, and we need to be consistent with the rest of the site.

Reply:
I actually copied this from a few lines above! But I see we don't make this mistake anywhere else so I've fixed it in both places.